io.github.sigmasd.VirtAudio.yml

```diff
@@ -0,0 +1,267 @@
+app-id: io.github.sigmasd.VirtAudio
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: virtaudio
+finish-args:
+  # Display
+  - --socket=wayland
+  - --socket=x11
```

Comment:
it's either `x11` or `wayland` + `fallback-x11`

Reply:
changed